packages/mikado_reborn/src/components/Checkbox/Checkbox.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div
+    class="mkr__checkbox"
+    role="checkbox"
+    :class="[{ 'checked' : value }, size]"
```

Comment:
Modifiers should follow the BEM convention.
```suggestion
    :class="[{ 'mkr__checkbox--checked' : value }, `mkr__checkbox--${size}`]"
```
Don't forget to update the sass rules according to this 😅